jenkins/spark-tests.sh

```diff
@@ -96,8 +96,8 @@ fi
 
 tar xzf "$RAPIDS_INT_TESTS_TGZ" -C $ARTF_ROOT && rm -f "$RAPIDS_INT_TESTS_TGZ"
 
-$MVN_GET_CMD -DremoteRepositories=$SPARK_REPO \
-    -DgroupId=org.apache -DartifactId=spark -Dversion=$SPARK_VER -Dclassifier=bin-hadoop3.2 -Dpackaging=tgz
+. jenkins/hadoop-def.sh $SPARK_VER
+wget -P $ARTF_ROOT https://archive.apache.org/dist/spark/spark-$SPARK_VER/spark-$SPARK_VER-$BIN_HADOOP_VER.tgz
```

Comment:
the same. At least in our CI, it would be nice to use internal artifact store. Would be better to use $SPARK_REPO to form the link